_includes/commands/image-cmds.md

```diff
@@ -0,0 +1,10 @@
+
+| Command  | Description                    |
+| -------- | ------------------------------ |
+| mceImage | Opens the insert image dialog. |
```

Comment:
Maybe for consistency say the `insert/edit image dialog`

Reply:
done